constants.py

```diff
@@ -7,3 +7,5 @@
 PROJECT_ROOT = Path(__file__).parent
 ASSETS_PATH = PROJECT_ROOT / 'tmp' / 'articles'
 CRAWLER_CONFIG_PATH = PROJECT_ROOT / 'scrapper_config.json'
+HEADERS = {'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, '
```

Comment:
nice! any better way to construct this dictionary to eliminate that strange line break?